ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/linear_optimisation/algorithms/fillers/MaxMinMarginFiller.java

```diff
@@ -94,21 +94,26 @@ private void buildMinimumMarginVariable(LinearProblem linearProblem) {
      * the margin of each Cnec. They consist in a linear equivalent of the definition
      * of the min margin : MM = min{c in CNEC} margin[c].
      * <p>
-     * For each Cnec c, the two constraints are (if the max margin is defined in MEGAWATT) :
+     * For each Cnec c, the 4 constraints are (if the max margin is defined in MEGAWATT) :
      * <p>
      * MM <= fmax[c] - F[c]    (ABOVE_THRESHOLD)
      * MM <= F[c] - fmin[c]    (BELOW_THRESHOLD)
+     * - (1 - P) * mMinRAM <= MM <= 0
+     * MM <= 0
      * <p>
-     * For each Cnec c, the two constraints are (if the max margin is defined in AMPERE) :
+     * For each Cnec c, the 4 constraints are (if the max margin is defined in AMPERE) :
      * <p>
      * MM <= (fmax[c] - F[c]) * 1000 / (Unom * sqrt(3))     (ABOVE_THRESHOLD)
      * MM <= (F[c] - fmin[c]) * 1000 / (Unom * sqrt(3))     (BELOW_THRESHOLD)
+     * - (1 - P) * mMinRAM <= MM
+     * MM <= 0
```

Comment:
This filler does not actually define the constraints with "P". These are only defined if the MaxMinRelativeMarginFiller is used. So you shouldn't update the comments here, you can do that in the relative filler